src/pages/Product.js

```diff
@@ -0,0 +1,36 @@
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+
+const Product = ({ data }) => {
+  const [isHover, setIsHover] = useState(false);
+  return (
+    <div
+      className="product"
+      key={data.id}
+      onMouseOver={() => {
```

Comment:
```suggestion
const Product = ({ data }) => {
  const [isHover, setIsHover] = useState(false);
  const {id, hover_image} = data;
  
  return (
    <div
      className="product"
      key={id}
      onMouseOver={() => {
```
구조분해할당을 사용하면 이렇게 사용할 수도 있습니다!